apps/angular-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "angular-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve -o",
+    "build": "ng build",
```

Comment:
I think it is better to remove the `build` command because we want to run it when we are going to publish the `packages` folder, so if any application inside the `apps` folder has a build command as well, it will consume some time that is not necessary to publish.

Reply:
Nvm. Check comment in the another PR